### DISCUSSION TITLE
Split animal charges in several categories

### DISCUSSION THREAD

**Discussion by Blipz** [Azgaar/Armoria#112] [Ideas]
If the Animals category becomes too crowded, we could split it like http://heraldicart.org/ does (Beasts, Reptiles, Invertebrates).

**Comment by Blipz** [comment]
Same remark about the Miscellaneous category (Tools, Arts, Clothing, Animal parts ?).

**Comment by Azgaar** [comment]
Sure, let's do it we have more charges there

**Comment by Blipz** [comment]
Done here: f28f8b1c3d9d6f34efc324b10f34be7989c22f55
